server/src/main/java/io/crate/protocols/postgres/Messages.java

```diff
@@ -107,7 +107,7 @@ static ChannelFuture sendCommandComplete(Channel channel, String query, long row
         buffer.writeByte('C');
         buffer.writeInt(length);
         writeCString(buffer, commandTagBytes);
-        ChannelFuture channelFuture = channel.writeAndFlush(buffer);
+        ChannelFuture channelFuture = channel.write(buffer);
         if (LOGGER.isTraceEnabled()) {
             channelFuture.addListener((ChannelFutureListener) future -> LOGGER.trace("sentCommandComplete"));
         }
```

Comment:
minor: can we have a misleading log message saying "sent" before the message got flushed?

Reply:
The futures only  trigger once the message is flushed out, so this is save. (We do the same already for most other messages)